### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: src/RestfulServer.php
Comment (sabina-talipova) [inline src/RestfulServer.php:1]: Open ticket:
https://github.com/silverstripe/silverstripe-elemental/issues/1114

File: tests/unit/RestfulServerTest.php
Comment (sabina-talipova) [inline tests/unit/RestfulServerTest.php:1]: Open ticket: https://github.com/silverstripe/silverstripe-restfulserver/issues/111

File: src/DataFormatter/JSONDataFormatter.php
Comment (sabina-talipova) [inline src/DataFormatter/JSONDataFormatter.php:1]: Open ticket: https://github.com/silverstripe/silverstripe-restfulserver/issues/112

File: tests/unit/JSONDataFormatterTest.php
Comment (sabina-talipova) [inline tests/unit/JSONDataFormatterTest.php:1]: Open ticket: https://github.com/silverstripe/silverstripe-restfulserver/issues/111

File: src/DataFormatter.php
Comment (sabina-talipova) [inline src/DataFormatter.php:1]: Open ticket: https://github.com/silverstripe/silverstripe-restfulserver/issues/113

File: src/RestfulServer.php [hunk 1]
```diff
@@ -22,23 +22,6 @@
...
- *       requested in this case)
- * @todo Custom filters per DataObject subclass, e.g. to disallow showing unpublished pages in
- * SiteTree/Versioned/Hierarchy
- * @todo URL parameter namespacing for search-fields, limit, fields, add_fields
- *       (might all be valid dataobject properties)
- *       e.g. you wouldn't be able to search for a "limit" property on your subclass as
- *       its overlayed with the search logic
- * @todo i18n integration (e.g. Page/1.xml?lang=de_DE)
- * @todo Access to extendable methods/relations like SiteTree/1/Versions or SiteTree/1/Version/22
- * @todo Respect $api_access array notation in search contexts
```
Comment (sabina-talipova) [inline src/RestfulServer.php:41]: - https://github.com/silverstripe/silverstripe-restfulserver/issues/111
- https://github.com/silverstripe/silverstripe-restfulserver/issues/112
- https://github.com/silverstripe/silverstripe-restfulserver/issues/113

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**